pkgs/development/python-modules/nanotime/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "nanotime";
+  version = "0.5.2";
+
+  src = fetchPypi {
```

Comment:
I see a test in the archive and the build output says they aren't being ran.
They say to use `nosetests`.

Reply:
Done but tests fail: https://github.com/jbenet/nanotime/issues/2

This is an really old package, but required as a dependency by dvc: https://github.com/NixOS/nixpkgs/issues/49438

So I just disable the tests for now.

Reply:
Sounds good.